assets/i18n/en.json

```diff
@@ -255,6 +255,7 @@
     "I18N_DELETE_ACCOUNT_PAGE_REDIRECT_INFO": "This will take you to a page where you can delete your Oppia account.",
     "I18N_DELETE_ACCOUNT_PAGE_TITLE": "Delete Account | Oppia",
     "I18N_DELETE_LEARNER_GROUP": "Delete Group",
+    "I18N_DEST_IF_STUCK_INFO_TOOLTIP": "You can now specify a new card in which you can walk the learners through the concepts used in the question, if they get really stuck!",
```

Comment:
Is this only shown in the editor page? We don't internationalize the editor pages, FYI.

Reply:
Ya, it's supposed to be shown only on the editor page. I'll fix this.